### DISCUSSION TITLE
Disable Fade In Feature

### DISCUSSION THREAD

**Discussion by PureBeatRadio** [AzuraCast/AzuraCast#6591] [Q&A]
Hello all, 

Newbie here to Azuracast:)

I have been slowly setting up my station with AC and so far so good:)

I do have a few issues, but the one I would like to fix is the option to disable "Fade In", as have noticed at the start of every piece of audio/song, it fades in at around 1.5sec?

Not really nice to hear from a radio station, as feel any audio starting should be instant and at normal volume.

I have noticed a few similar questions on here, but think most are to do with when AC goes from Auto DJ to Live.

I have mostly uploaded pre recorded shows/audio on my AC playlist/schedule, so no Live broadcast as such. 

At present, I do have Crossfade enabled (3.7sec), as do like its smooth transition, but not when a song gets faded in, hence why I would love to have Fade In disabled.

I do note that on the Liquidsoap configuration, I have the following code. Can it be changed to have zero fade in?

**def live_aware_crossfade(old, new) =
    if !to_live then
        # If going to the live show, play a simple sequence
        sequence([fade.out(old.source),fade.in(new.source)])
    else
        # Otherwise, use the smart transition
        cross.simple(old.source, new.source, fade_in=3.70, fade_out=3.70)
    end
end**

radio = cross(minimum=0., duration=5.55, live_aware_crossfade, radio)

Any help on this matter is greatly appreciated.....Thanks, Russ

**Comment by BusterNeece** [comment]
Not sure if you're still awaiting an answer on this, but I'd suspect the short answer is that if you disable crossfading on our end, and then edit the custom Liquidsoap configuration to put your custom crossfade code next to where the original code would go, it should solve this.
